typescript/db/src/resolvers/workspace.ts

```diff
@@ -103,6 +110,43 @@ const createWorkspace = async (
     },
     user
   );
+  if (createdWorkspaceId) {
+    const now = new Date();
+    const workspaceSlug = getSlug(args.data.name);
+    const imagesToCreate = tutorialImages.map(
+      ({ name, url, width, height, mimeType }, urlIndex) => {
+        const createdAt = new Date();
+        createdAt.setTime(now.getTime() + urlIndex);
+
+        return {
+          externalUrl: url,
+          name,
+          createdAt: createdAt.toISOString(),
+          noThumbnails: true,
+          width,
+          height,
+          mimeType,
+        };
+      }
+    );
+    const labels = tutorialLabels.map((label) => ({
+      ...omit(["id", "imageId"], label),
+    }));
+    const labelClass = tutorialLabelClasses[0];
```

Comment:
why have an array if we use only one? either change `dataset-tutorial.ts` to only have a single label class without an array, or make the rest of the code fully support the possibility of having multiple label classes.

Reply:
Again, I reused the code already present.